app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true, length: { maximum: 30 }
+  validates :body, presence: true, length: { maximum: 255 }
+  validates :author, presence: true, length: { minimum: 4 }
```

Comment:
Pls see my comment on the schema. We won't need to have a column for author anymore as we are already associating post with `user`.